File: 0001-two-sum/0001-two-sum.java
class Solution {
    // The following is my solution to this problem.
    public int[] twoSum(int[] nums, int target) {
        int answer[] =new int[2];
        for(int i=0;i<=nums.length-1;i++){
            for(int j=1;j<=nums.length-1;j++){
                if((nums[i]+nums[j]) == target && i!=j){
                    answer[0]=i;
                    answer[1]=j;
                    return answer;
                }
            }
        }
       return answer; 
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        target_map = {}
        result = []
        for num in range(len(nums)):
            if target-nums[num] in target_map:
                result.append(nums.index(target-nums[num]))
                result.append(num)
                return result
            elif nums[num] not in target_map:
                target_map[nums[num]] = 1
            else:
                target_map[nums[num]] += 1
        